orchestra/models.py

```diff
@@ -559,6 +560,35 @@ class TaskTimer(models.Model):
     stop_time = models.DateTimeField(null=True)
 
 
+class PayRate(models.Model):
+    """
+    A PayRate object tracks how much a worker is paid over time.
+
+    Attributes:
+        worker (orchestra.models.Worker):
+            The worker who is being paid.
+        hourly_rate (djmoney.models.fields.MoneyPatched):
+            The amount of money the worker receives per hour.
+        hourly_multiplier (decimal.Decimal):
+            Any multiplier that gets applied to a worker's pay
+            (e.g., fees imposed by a payment gateway).
+        start_datetime (datetime.datetime):
+            The beginning of the time period of the pay rate.
+        end_datetime (datetime.datetime):
+            The end of the time period of the pay rate,
+            or None if it's the current period.
+    """
+    worker = models.ForeignKey(Worker, related_name='pay_rates')
+    hourly_rate = MoneyField(max_digits=10, decimal_places=2, default_currency='USD')
+    multiplier = models.DecimalField(max_digits=6, decimal_places=4)
```

Comment:
chatted offline---we couldn't figure out a clean way to represent the fixed rate (is it per transaction, or per unit transfer amount, or ?), so we're leaving it out of this one.